challenge_3/c/scottsquatch/src/main.c

```diff
@@ -0,0 +1,94 @@
+#include <stdlib.h>
+#include <stdio.h>
+
+/** Constants **/
+#define MODE_READ  "r"
+#define NUM_ARGS 3
+#define FILE_NAME_ARG 1
+#define ARRAY_SIZE_ARG 2
+
+/** Function Prototypes **/
+int int_compare(const void*, const void*);
+void print_array(int*, int);
+int get_majoriy_element(int*, int);
+int* read_array(char*,int);
+
+int main(int argc, char* argv[])
+{
+  // Get parameters from input
+  if (argc != NUM_ARGS)
+  {
+    // Incorrect number of parameters
+    printf("Incorrect Number of parameters, expected %d but got %d\n",
```

Comment:
If instead you would like to print to stderr to indicate error messages, use `fprintf(stderr, "`.